src/WooCommerce.php

```diff
@@ -395,6 +409,17 @@ public static function getProductDetailsHtmlDataAttr(\WC_Product $product, $is_d
       }
     }
 
+    if (!isset($product_details['variant']) && $product->get_type() === 'variation') {
```

Comment:
Please move the `$is_detail_view` condition of the preceding condition into an inner condition and then turn your added code into the else case of that condition:
```suggestion
    elseif (!isset($product_details['variant'])) {
```

Reply:
This is not a detail view.

Reply:
This is on order confirmation.